webapp/src/components/channel_link_label/component.tsx

```diff
@@ -47,14 +47,17 @@ const ChannelLinkLabel = (props: Props) => {
                     {
                         display: 'flex',
                         alignItems: 'center',
-                        marginLeft: 'auto',
+                        justifyContent: 'center',
                         height: 'auto',
+                        minWidth: '20px',
+                        marginLeft: 'auto',
+                        marginRight: '4px',
                     }
                 }
             >
                 <ActiveCallIcon
                     fill={props.theme.sidebarText}
-                    style={{width: '12px', height: '12px'}}
+                    style={{minWidth: '12px', maxWidth: '12px', minHeight: '12px', maxHeight: '12px'}}
```

Comment:
I'm assuming that something is making the width and height not work correctly?

Reply:
@cpoile Not exactly sure since I couldn't reproduce but there was an instance in which the icon would collapse:

![image](https://github.com/mattermost/mattermost-plugin-calls/assets/1832946/12c7d37a-7b73-4bd4-877b-5c6c32da69c4)